tests/Features/Expect/toHaveProperties.php

```diff
@@ -2,7 +2,7 @@
 
 use PHPUnit\Framework\ExpectationFailedException;
 
-test('pass', function () {
+test('pass objects', function () {
```

Comment:
I don't think we should transform the tests, but rather create new ones. Does this perhaps make way for: `toHavePropertyObject` with all that logic instead of changing `toHaveProperty`?